target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -329,6 +329,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch1" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
 		;;
+	engenius,esr600)
+	    ucidef_add_switch "switch0" \
+			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5:wan" "0t@eth0"
```

Comment:
Does it really need to be "0t@eth0" here instead of "0@eth0"? I thought both would give the same result?

Reply:
The answer that it appears to work equally well with either "0t@eth0" or "0@eth0".
Since "0@eth0" appears to be the preferred notation I'll switch to that.